docs/medical-api/more-info/medical-record-summary.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Medical Record Summary"
+description: "Consolidate Medical Records Into a Single Document"
+---
+
+This guide will teach you how to generate a Medical Record Summary, which condenses all of patients' medical records into a single document.
+
+To do this, Metriport extracts FHIR resources across all available CDA documents for a patient and de-duplicates them, resulting in a set of unique medical records, which are then written into HTML and PDF files, making them extremely easy to navigate.
+
+Here's the title page of an example Medical Record Summary:
+
+<img className="h-100" src="/images/example-medical-record.png" />
+
+## Generate using API
+
+To generate a Medical Record Summary using Metriport API, you need to follow these steps:
+
+1. Start a [Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post). This endpoint allows you to choose [FHIR resources](/medical-api/fhir/overview), date range, and rendering (`PDF` or `HTML`) for your Medical Record Summary. When this query completes, a [webhook](/medical-api/more-info/webhooks) with a JSON payload containing all these resources will be sent to you.
+
+<Tip>
+  Once the query is kicked off, you can monitor its status using the [Get Consolidated Data Query
+  Status](/medical-api/api-reference/fhir/consolidated-data-query-get) endpoint.
+</Tip>
+
+2. In addition to webhooks, you can download the Medical Record Summary in a PDF or HTML format using the [Get Medical Record Summary](/medical-api/api-reference/fhir/get-medical-record-summary) endpoint.
+
+<Tip>
+  Prior to creating or downloading a new Medical Record Summary, you can check when it was created,
+  and whether it's available in either PDF or HTML format using the [Get Medical Record Summary
+  Status](/medical-api/api-reference/fhir/get-medical-record-summary-status) endpoint.
+</Tip>
```

Comment:
I'd consider taking this out entirely to not encourage devs to poll our API for when an MR summary becomes available

The main reason we have this anyways is to support our Dash